src/ResourceManager/DevSpaces/Commands.DevSpaces.Test/ScenarioTests/DevSpacesTests.ps1

```diff
@@ -0,0 +1,74 @@
+<#
+.SYNOPSIS
+Test DevSpaces stuff
+#>
+function Test-AzureRmDevSpacesController
+{
+    # Setup
+    $resourceGroupName = Get-RandomResourceGroupName
+    $kubeClusterName = Get-RandomClusterName
+    $devSpacesName = Get-DevSpacesControllerName
+	
+	$devSpaceController = New-AzureRmDevSpacesController -ResourceGroupName $resourceGroupName -Name $devSpacesName -TargetClusterName $kubeClusterName -TargetResourceGroupName $resourceGroupName
+
+	Assert-AreEqual $devSpacesName $devSpaceController.Name
+	Assert-AreEqual $resourceGroupName $devSpaceController.ResourceGroupName
+	Assert-AreEqual "Succeeded" $devSpaceController.ProvisioningState
+	Assert-AreEqual "eastus" $devSpaceController.Location
```

Comment:
if you can, use the [Get-Location](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Common.ps1#L583) function to get the location that the DevSpacesController will be created at.

Reply:
DevSpaces are created at the same location as Kubernetes cluster. I have assumed in the test case that AKS cluster exists in eastus.

Reply:
Is the test not creating the AKS cluster? If so, would anyone recording the tests need to create one?

We generally recommend that a test create and tear down any resources it needs for it to be successfully recorded. I would recommend the same here.

Reply:
The user doesn't have to create AKS cluster. The rest API calls to DevSpaces are mocked. So Create DevSpaces will succeed if arguments are passed as expected. No real AKS cluster is created.

Reply:
Rest apis are mocked here https://github.com/Azure/azure-powershell/pull/6449/files#diff-9b9a7f0c5c3cc9e9ccbdfb3618aa9e72

Reply:
Done